transformers/integration_tests/flip_file_cat_test.go

```diff
@@ -0,0 +1,54 @@
+package integration_tests
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/events/flip_file/cat"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/libraries/shared/fetcher"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("FlipFileCat Transformer", func() {
+	BeforeEach(func() {
+		test_config.CleanTestDB(db)
+	})
+
+	flipFileCatConfig := event.TransformerConfig{
+		TransformerName:   constants.FlipFileCatTable,
+		ContractAddresses: test_data.FlipV110Addresses(),
+		ContractAbi:       constants.FlipV110ABI(),
+		Topic:             constants.FlipFileCatSignature(),
+	}
+
+	XIt("fetches and transforms a Flip File Cat event", func() {
```

Comment:
I think that there was a cat file event at 10769102, so we can probably un-pend this integration test.

Reply:
I didn't see anything on the `MCD_FLIP_BAT_A_1.1.0`, `MCD_FLIP_ETH_A_1.1.0`, `MCD_FLIP_KNC_A_1.1.0`, or `MCD_FLIP_PAXUSD_A_1.1.1` contracts. I've been assuming this event would be from calling `function file(bytes32 what, address data)` where `what = "cat"` on one of those `Flip` contracts

Reply:
Doh, you're right. I was looking at the wrong contract. Sorry about that!